src/pages/trainer/training/edit/components/editor.tsx

```diff
@@ -0,0 +1,49 @@
+import * as React from 'react';
+import {ToolbarContainerComponent} from './toolbar';
+const classNames: any = require('./editorStyles.scss');
+
+interface IProps {
+  content: string;
+  cursorStartPosition: number;
+  shouldSetEditorFocus: boolean;
+  className: string;
```

Comment:
With this className property I'm exposing to outside the possibility to change styles for this Editor. Like other components where you can add css clases